rust/Cargo.toml

```diff
@@ -27,6 +27,7 @@ chrono = "0"
 uuid = { version = "0.8", features = ["serde", "v4"] }
 lazy_static = "1"
 percent-encoding = "2"
+time = "0.1"
```

Comment:
why do we need that?

Reply:
This is for the time duration at `checkpoint.rs` L308.

```rust
let updated = (
  current_file.0,
  ObjectMeta {
    path: current_file.1.path.clone(),
    modified: last_file.1.modified.add(Duration::seconds(1)),
  },
);
```

Reply:
yeah, but I believe we can use `core::time::Duration`  or even `chrono`'s one

Reply:
Ah, let me try it.

Reply:
`chrono::Duration` works but `core::time::Duration` doesn't. I replaced it with `chrono::Duration`. Thanks.